src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -1625,6 +2019,36 @@ bool Pane::FocusPane(const uint32_t id)
     return false;
 }
 
+// Method Description:
+// - Recursive function that finds a pane with the given ID
+// Arguments:
+// - The ID of the pane we want to find
+// Return Value:
+// - A pointer to the pane with the given ID, if found.
+std::shared_ptr<Pane> Pane::FindPane(const uint32_t id)
+{
+    if (_IsLeaf())
+    {
+        if (id == _id)

```

Comment:
I did it this way because having the conjunction of predicates means that a pane that is a leaf, but doesn't have the right ID would go into the `else` block, but it doesn't have children and so it would crash.